lib/fern/api.rb

```diff
@@ -25,4 +54,4 @@ def endpoint(name, &block)
   end
 end
 
-ActiveSupport.on_load(:action_controller) { include Fern::Api }
+# ActiveSupport.on_load(:action_controller) { include Fern::Api }
```

Comment:
Bummer this didn't work; I sorta liked it. We should probably just remove it instead of commenting it out though.

Reply:
I think it _could_ work, with some thought -- the issue is that when you have other controllers that inherit `ActionController` that do work on params `params` (like Devise). 

I don't know much about `on_load` -- I was thinking maybe, instead of having to define `FernController` on the project level, have a base `FernController` class defined in this gem?

Reply:
I definitely support the idea of doing as little work as possible to include this in your project